ui/app/src/components/PreviewAddressBar/PreviewAddressBar.tsx

```diff
@@ -121,6 +121,16 @@ export function PreviewAddressBar(props: AddressBarProps) {
   const [openSelector, setOpenSelector] = useState(false);
   const [focus, setFocus] = useState(false);
   const dispatch = useDispatch();
+  const rootPath = '/site/website/index.xml';
```

Comment:
When the item is not a page, the selected item won't be sent to the SingleItemSelector (selectedItem is only used by the SingleItemSelector). The 'not page' logic is only for this usage in PreviewAddressBar, since the SingleItemSelector is used in other places with different rootPath values.